AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -467,10 +467,8 @@ - (void)setPlayerState:(ASVideoNodePlayerState)playerState
 
 - (void)setAssetURL:(NSURL *)assetURL
 {
-  ASDN::MutexLocker l(__instanceLock__);
-
   if (ASObjectIsEqual(assetURL, self.assetURL) == NO) {
```

Comment:
This change allows there to be a small possibility that setAndFetchAsset will be called when assetURL and self.assetURL are equal (because the lock is released after the check). Is that safe?